packages/runner/lib/exec.ts

```diff
@@ -277,6 +285,7 @@ export async function exec(
                 };
             }
         } finally {
+            await nango.releaseAllLocks().catch((err: unknown) => logger.warning('Failed to release all locks', { reason: err }));
```

Comment:
await and catch is confusing imo, can you use a try/catch?

Reply:
sure. I thought for logging only it was simple enough